src/runtime/utils/utils.ts

```diff
@@ -23,13 +23,13 @@ export function isNumber(value: any): boolean {
   return typeof value === 'number';
 }
 
-export function forEach(obj: object, iteratee: (value, key) => void) {
+export function forEach(obj: object, iteratee: (value: any, key: string) => void) {
   Object.keys(obj).forEach(key => iteratee(obj[key], key));
 }
 
 export function reduceObj(
   obj: object,
-  iteratee: (acc, currentValue: {key: string; value: any}, index?: number) => object
+  iteratee: (acc: object, currentValue: {key: string; value: any}, index?: number) => object
```

Comment:
added general `object` type, cause it's not vital to specify more strict typing here. left `value: any`, as it was already in the codebase. Non vital actually.